src/controllers/roomController.js

```diff
@@ -0,0 +1,110 @@
+import models from '../models';
+import { successResponse, errorResponse } from '../utils';
+
+const { Rooms, Accommodations } = models;
+/**
+ * @class RoomsController
+ * @description Controllers for handling rooms requests
+ * @exports AccommodationController
+ */
+class RoomController {
+  /**
+    * @method createRooms
+    * @description Method to create room facilities
+    * @param {object} req - The Request Object
+    * @param {object} res - The Response Object
+    * @returns {object} Newly created room details
+    */
+  static async createRoom(req, res) {
+    const {
+      type, capacity, image, accommodationId
+    } = req.body;
+    try {
+      const checkAccommodation = await Accommodations.findOne({
+        where: { id: accommodationId }
+      });
+      if (!checkAccommodation) {
+        return errorResponse(res, 404, 'Accommodation does not exist');
+      }
+      const result = await Rooms.create({
+        type, capacity, image, accommodationId
+      });
+      return successResponse(res, 202, 'Room created', result);
+    } catch (error) {
+      return errorResponse(res, 500, 'Error creating room');
+    }
+  }
+
+  /**
+    * @method updateRoom
+    * @description Method to update room facilities
+    * @param {object} req - The Request Object
+    * @param {object} res - The Response Object
+    * @returns {object} Update room details
+    */
+  static async updateRoom(req, res) {
+    const { roomId } = req.params;
+    const {
+      type, capacity, image, accommodationId
```

Comment:
Since updating a room does not change the `accommodationId`, there is no need to send the `accommodationId` when updating a room